roles/cms/tasks/copy_config.yml

```diff
@@ -1,22 +1,22 @@
-- name: Copy cms.conf file
+- name: Copy cms.conf file to home directory
   become: yes
   template:
-    src: cms.conf.j2
-    dest: /usr/local/etc/cms.conf
+    src: cms.toml.j2
+    dest: "/home/cmsuser/cms/etc/cms.toml"
     owner: cmsuser
     group: cmsuser
-    mode: '0660'
+    mode: "0660"
```

Comment:
Upstream installation scripts stopped producing group-writable directories, so the automation should probably follow.